packages/edit-post/src/components/sidebar/plugin-post-excerpt/test/index.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+
+/**
+ * WordPress dependencies
+ */
+import { SlotFillProvider } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import PluginPostExcerptPanel from '../';
+
+describe( 'PluginPostExcerptPanel', () => {
+	test( 'renders fill properly', () => {
+		const { container } = render(
+			<SlotFillProvider>
+				<PluginPostExcerptPanel className="my-plugin-post-excerpt-custom-content">
+					Post Excerpt - Custom content
+				</PluginPostExcerptPanel>
+				<PluginPostExcerptPanel.Slot />
+			</SlotFillProvider>
+		);
+
+		expect( container ).toMatchSnapshot();
+	} );
+} );
```

Comment:
I'd say snapshot tests bring little value here. Should we add a couple of more detailed assertions here? For example:

```suggestion
/**
 * External dependencies
 */
import { render, screen } from '@testing-library/react';

/**
 * WordPress dependencies
 */
import { SlotFillProvider } from '@wordpress/components';

/**
 * Internal dependencies
 */
import PluginPostExcerptPanel from '../';

describe( 'PluginPostExcerptPanel', () => {
	test( 'renders fill properly', () => {
		render(
			<SlotFillProvider>
				<PluginPostExcerptPanel className="my-plugin-post-excerpt-custom-content">
					Post Excerpt - Custom content
				</PluginPostExcerptPanel>
				<div role="tabpanel">
					<PluginPostExcerptPanel.Slot />
				</div>
			</SlotFillProvider>
		);

		expect( screen.getByRole( 'tabpanel' ) ).toHaveTextContent(
			'Post Excerpt - Custom content'
		);
		expect(
			screen.getByText( 'Post Excerpt - Custom content' )
		).toHaveClass( 'my-plugin-post-excerpt-custom-content' );
	} );
} );
```

Reply:
accepting the commit and then updating the code